spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/KerasUtils.scala

```diff
@@ -106,4 +108,46 @@ object KerasUtils {
     }
   }
 
+  private[keras] def toBigDLCriterion[T : ClassTag](loss: String)
+    (implicit ev: TensorNumeric[T]): Criterion[T] = {
+    loss.toLowerCase() match {
+      case "categorical_crossentropy" => CategoricalCrossEntropy[T]()
```

Comment:
we should move the unittest from python to scala as well with the mapping logic

Reply:
Newly added criterions already have unit tests. I will add the old ones in Scala.